meta-genivi-dev/recipes-sota/python-pygobject/python-pygobject_2.28.3.bb

```diff
@@ -0,0 +1,55 @@
+# Copied from http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-devtools/python?h=jethro d462015
```

Comment:
Honestly, I hadn't spotted the poky/meta layer in there, I had only seen the meta-* layers in the root directory. However, I tried removing this python-pygobject_2.28.3.bb and using the python-pygobject_3.18.2 from poky, and from the python prompt I was unable to run `import gobject`.

```
root@qemux86-64:~# python
Python 2.7.11 (default, Jan 27 2017, 13:16:50) 
[GCC 5.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import gobject
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named gobject
```

So I guess something else has changed with the upgrade?